### DISCUSSION TITLE
4G COTS UE with srsRAN Project

### DISCUSSION THREAD

**Discussion by Monsieur-ugur** [srsran/srsRAN_Project#692] [General Help] [answered]
Hello everyone, I haven't seen anyone talk about it on this forum but is it possible to connect a COTS UE using 4G (smartphone) with a srsRAN Project gNB (a USRP). If someone did it does he have some tips?

Whishin you a nice day.

**Comment by pgawlowicz** [comment, marked as answer]
no, srsRAN Project gNB supports only 5G. 
You should use srsenb from [srsRAN-4G repo](https://github.com/srsran/srsRAN_4G)

**Reply by Monsieur-ugur** [reply]
Thank you very much, have a nice day.
